src/renderer/pages/app.js

```diff
@@ -1,32 +1,36 @@
 const colors = require('material-ui/styles/colors')
+const createGetter = require('fn-getter')
 const React = require('react')
 
 const darkBaseTheme = require('material-ui/styles/baseThemes/darkBaseTheme').default
-const lightBaseTheme = require('material-ui/styles/baseThemes/lightBaseTheme').default
 const getMuiTheme = require('material-ui/styles/getMuiTheme').default
 const MuiThemeProvider = require('material-ui/styles/MuiThemeProvider').default
 
 const Header = require('../components/header')
+const TorrentListPage = require('./torrent-list-page')
 
 const Views = {
-  'home': require('./torrent-list-page'),
-  'player': require('./player-page'),
-  'create-torrent': require('./create-torrent-page'),
-  'preferences': require('./preferences-page')
+  'home': createGetter(() => TorrentListPage),
+  'player': createGetter(() => require('./player-page')),
+  'create-torrent': createGetter(() => require('./create-torrent-page')),
+  'preferences': createGetter(() => require('./preferences-page'))
```

Comment:
Same as controllers, lazy loading the views means that we don't need to inline all the require calls within each view by hand.